mods/cnc/weapons/largecaliber.yaml

```diff
@@ -1,50 +1,19 @@
 70mm:
+	Inherits: ^BallisticWeapon
 	ReloadDelay: 30
 	Range: 4c0
 	Report: tnkfire3.aud
 	Projectile: Bullet
-		Image: 120MM
 		Speed: 853
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 25
 		Versus:
 			None: 25
 			Wood: 75
 			Light: 100
 			Heavy: 90
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
-	Warhead@3Eff: CreateEffect
-		Explosions: small_frag
-		ImpactSounds: xplos.aud
 
-105mm:
-	ReloadDelay: 50
-	Range: 4c768
-	Report: tnkfire4.aud
-	Projectile: Bullet
-		Image: 120MM
-		Speed: 682
-	Warhead@1Dam: SpreadDamage
-		Spread: 128
-		Damage: 30
-		Versus:
-			None: 30
-			Wood: 75
-			Light: 75
-			Heavy: 100
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
-	Warhead@3Eff: CreateEffect
-		Explosions: small_frag
-		ImpactSounds: xplos.aud
-
-120mm:
+^BallisticWeapon:
```

Comment:
Yes, I only did that to keep the diff as small as possible for reviewability. Fixed now.